Game Dev/Assets/Scripts/KillingSlider/KillingSlider.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
```

Comment:
ar trebui sa implementam si un behaviour care sa dea trigger la metrou. Ex: daca playerul sta nemiscat mai mult de 15 secunde -> moare, daca nu termina nivelul in 2 min -> moare